trustpilot/async_client.py

```diff
@@ -1,21 +1,9 @@
 try:
     import aiohttp
 except ImportError:
-    raise RuntimeError("aiohttp is not installed, possibly running in python27?")
-
-import sys
-
-version_info = sys.version_info
-if version_info.major == 3 and version_info.minor < 7:
-    try:
-        from async_generator import asynccontextmanager
-    except:
-        raise RuntimeError(
-            "You are using python<3.7 and have not install async_generator"
-        )
-else:
-    from contextlib import asynccontextmanager
+    raise RuntimeError("aiohttp is not installed")
```

Comment:
aiohttp is now a direct dependency so it will always be installed, no need for try except anymore

Reply:
Fixed